src/style.css

```diff
@@ -1,3 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: black;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  width: 1200px;
```

Comment:
My previous comment is related to this one. Removing flex from the body will make setting width unnecessary.
This approach with fixed width value is not ideal for [RWD](https://pl.wikipedia.org/wiki/Responsive_web_design)